cmd/application.go

```diff
@@ -167,6 +172,38 @@ var applicationSetCmd = &cobra.Command{
 	},
 }
 
+var applicationDescribeCmd = &cobra.Command{
+	Use:   "describe",
+	Short: "describe the given application",
+	Run: func(cmd *cobra.Command, args []string) {
+		client := getOcClient()
+		fmt.Println("Application", args[0], " has:")
+
+		a, _ := component.List(client)
```

Comment:
you need to check for errors, here and in other places.

But otherwise, this looks good.

Reply:
yeah